src/main/java/com/tenx/ms/retail/order/rest/dto/OrderProduct.java

```diff
@@ -0,0 +1,26 @@
+package com.tenx.ms.retail.order.rest.dto;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@ApiModel("Order Product")
+@Data
+public class OrderProduct {
+    @ApiModelProperty(name = "Order product id", readOnly = true)
+    private Long orderProductId;
+
+    @ApiModelProperty(name = "Order Id")
+    private Long orderId;
+
+    @ApiModelProperty(name = "Product Id", required = true)
+    private Long productId;
+
+    @ApiModelProperty(name = "Order Product count", required = true, example = "10")
```

Comment:
@marielbysoares from the dto standpoint, IMO it looks cleaner not to have orderId, as it is implicit from the parent order.